.vitepress/theme/components/VueJobs.vue

```diff
@@ -1,14 +1,29 @@
 <script lang="ts">
 // shared data across instances so we load only once
 const base = `https://app.vuejobs.com/feed/vuejs/docs?format=json`
-let items = $ref([])
+
 </script>
 
 <script setup lang="ts">
 import { onMounted, computed } from 'vue'
 
 let vuejobs = $ref<HTMLElement>()
 
+let items = $ref<Jobs[]>([])
```

Comment:
As noted in the comment at the start of the `<script>` section, `items` needs to be in that section to ensure the data only gets loaded once. We lose that if we move it into the `<script setup>`.